3.5/sdk/windowsservercore-1803/Dockerfile

```diff
@@ -42,11 +42,11 @@ RUN `
     && rmdir /S /Q "%ProgramData%\Package Cache"
 
 # Install .NET 4.8 SDK
-RUN curl -fSLo sdk_tools48.zip https://dotnetbinaries.blob.core.windows.net/dockerassets/sdk_tools48.zip `
-    && tar -zxf sdk_tools48.zip `
-    && start /w msiexec /i sdk_tools48.msi /quiet VSEXTUI=1 `
-    && del sdk_tools48.msi `
-    && del sdk_tools48.zip `
+RUN mkdir sdk_tools48 `
+    && curl -fSLo sdk_tools48/sdk_tools48.zip https://dotnetbinaries.blob.core.windows.net/dockerassets/sdk_tools48.zip `
```

Comment:
From a consistency standpoint, would it make sense to use `\` in the paths consistently?  e.g. line 46 and 47?